src/main/java/frc/robot/Robot.java

```diff
@@ -20,38 +23,47 @@
  * the package after creating this project, you must also update the build.gradle file in the
  * project.
  */
+
+
+
+
+
 public class Robot extends TimedRobot {
   private Command autoCommand;
   public static ShuffleBoardManager shuffleboard;
   public static Drivetrain drive;
+  public static DriveSub driveSub;
   // TODO 2.3.9: Create variable for your subsystem
   public static BaseDriverConfig driver;
   public static Operator operator;
-
   private static boolean isTestMode = false;
 
   /**
    * This function is run when the robot is first started up and should be used for any
    * initialization code.
    */
+
   @Override
   public void robotInit() {
 
+    driveSub = new DriveSub(DriveConstants.SUB_SYSTEM_MOTOR);
+
     // This is really annoying so it's disabled
     DriverStation.silenceJoystickConnectionWarning(true);
 
     // make subsystems
     drive = new Drivetrain();
     // TODO 2.3.10: Create your subsystem
     
-    shuffleboard = new ShuffleBoardManager(drive);
+    shuffleboard = new ShuffleBoardManager(drive, driveSub);
```

Comment:
Your constructor definition in the ShuffleboardManager only takes in one parameter.

Reply:
It takes 2:
```java
public ShuffleBoardManager(Drivetrain drive, DriveSub driveSub){
```